### PR TITLE
Add org.freedesktop.LinuxAudio.Plugins.ZynFusion

### PR DESCRIPTION
**Pull request by takuvata**

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.LinuxAudio.Plugins.ZynFusion

**Comment by hfiguiere** [comment]
I forgot, to pass on flathub you need to add a `flathub.json` file with the following content

```json
{
    "skip-icons-check": true
}
```

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.LinuxAudio.Plugins.ZynFusion

File: org.freedesktop.LinuxAudio.Plugins.ZynFusion.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    "runtime": "org.freedesktop.LinuxAudio.BaseExtension",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk//19.08",
+    "build-extension": true,
+    "appstream-compose": false,
+    "cleanup": [
+        "/bin/pyth*",
+        "/bin/pip*",
+        "/bin/*.py",
+        "/lib/python2.7"
```
[the remark this answers is not in the document]
Comment (takuvata) [reply org.freedesktop.LinuxAudio.Plugins.ZynFusion.json]: Cleaning up lv2 and libpython.

No, nothing python is needed by the plugin beyond build stage.

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.LinuxAudio.Plugins.ZynFusion

**Comment by hfiguiere** [comment]
Today I discovered that apparently extension don't stripped. This one weight 185MB, but once I strip it, it weights ~59MB.

With the following patch.

```diff
diff --git a/org.freedesktop.LinuxAudio.Plugins.ZynFusion.json b/org.freedesktop.LinuxAudio.Plugins.ZynFusion.json
index 07480f5..103ddae 100644
--- a/org.freedesktop.LinuxAudio.Plugins.ZynFusion.json
+++ b/org.freedesktop.LinuxAudio.Plugins.ZynFusion.json
@@ -79,8 +79,11 @@
                 "install src/osc-bridge/schema/test.json $FLATPAK_DEST/lib/schema",
                 "install -d $FLATPAK_DEST/lib/font",
                 "install -D deps/nanovg/example/*.ttf $FLATPAK_DEST/lib/font",
+                "strip mruby/bin/mruby",
                 "install mruby/bin/mruby $FLATPAK_DEST/lib",
+                "strip libzest.so",
                 "install libzest.so $FLATPAK_DEST/lib",
+                "strip zest",
                 "install zest $FLATPAK_DEST/lib"
             ],
             "sources": [
@@ -121,6 +124,8 @@
             ],
             "post-install": [
                 "mv ${FLATPAK_DEST}/vst ${FLATPAK_DEST}/lxvst",
+                "strip ${FLATPAK_DEST}/lxvst/*.so",
+                "strip ${FLATPAK_DEST}/lv2/*.lv2/*.so",
                 "install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo org.freedesktop.LinuxAudio.Plugins.ZynFusion.metainfo.xml",
                 "appstream-compose --basename=org.freedesktop.LinuxAudio.Plugins.ZynFusion --prefix=${FLATPAK_DEST} --origin=flatpak org.freedesktop.LinuxAudio.Plugins.ZynFusion",
                 "install -Dm644 -t $FLATPAK_DEST/share/licenses/zynadsubfx/ COPYING"
```

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.LinuxAudio.Plugins.ZynFusion

**Comment by hfiguiere** [comment]
To the flathub maintainers: this plugin look great on my side !

Thanks !

**Pull request closed without merging**